examples/secondary-care/community/repeat-dispensing/nominated-pharmacy/clinical-practitioner/single-medication-request/1-Convert-Response-Dispense-200_OK.xml

```diff
@@ -0,0 +1,207 @@
+        <part typeCode="PART">
+          <partSDSRole classCode="ROL">
+            <id extension="UNKNOWN" root="1.2.826.0.1285.0.2.1.104"/>
+          </partSDSRole>
+        </part>
+      </AgentPersonSDS>
+    </author>
+    <author1 typeCode="AUT">
+      <AgentSystemSDS classCode="AGNT">
+        <agentSystemSDS classCode="DEV" determinerCode="INSTANCE">
+          <id extension="200000001285" root="1.2.826.0.1285.0.2.0.107"/>
+        </agentSystemSDS>
+      </AgentSystemSDS>
+    </author1>
+    <subject>
+      <DispenseNotification classCode="INFO" moodCode="EVN">
+        <id root="0798CE5E-B84E-4749-A9AB-B198F7445315"/>
+        <code code="163541000000107" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Dispensed Medication - FocusActOrEvent (administrative concept)"/>
+        <effectiveTime value="20210902155537"/>
+        <typeId extension="PORX_MT024001UK31" root="2.16.840.1.113883.2.1.3.2.4.18.7"/>
+        <recordTarget typeCode="RCT">
+          <patient classCode="PAT">
+            <id extension="9449305552" root="2.16.840.1.113883.2.1.4.1"/>
+          </patient>
+        </recordTarget>
+        <primaryInformationRecipient contextControlCode="ON" typeCode="PRCP">
+          <AgentOrg classCode="AGNT">
+            <agentOrganization classCode="ORG" determinerCode="INSTANCE">
+              <id extension="VNFKT" root="1.2.826.0.1285.0.1.10"/>
+              <code code="999" codeSystem="2.16.840.1.113883.2.1.3.2.4.17.94"/>
+              <name>FIVE STAR HOMECARE LEEDS LTD</name>
+            </agentOrganization>
+          </AgentOrg>
+        </primaryInformationRecipient>
+        <pertinentInformation1 contextConductionInd="true" typeCode="PERT">
+          <templateId extension="CSAB_RM-NPfITUK10.pertinentInformation" root="2.16.840.1.113883.2.1.3.2.4.18.2"/>
+          <pertinentSupplyHeader classCode="SBADM" moodCode="EVN">
+            <id root="0798CE5E-B84E-4749-A9AB-B198F7445315"/>
+            <code code="225426007" codeSystem="2.16.840.1.113883.2.1.3.2.4.15"/>
+            <effectiveTime nullFlavor="NA"/>
+            <author contextControlCode="OP" typeCode="AUT">
+              <time value="20210707154300"/>
+              <signatureText nullFlavor="NA"/>
+              <AgentPerson classCode="AGNT">
+                <id extension="999999999999" root="1.2.826.0.1285.0.2.0.67"/>
+                <code code="R9999" codeSystem="1.2.826.0.1285.0.2.1.104"/>
+                <telecom use="WP" value="tel:01234567890"/>
+                <agentPerson classCode="PSN" determinerCode="INSTANCE">
+                  <id extension="G9999999" root="1.2.826.0.1285.0.2.1.54"/>
+                  <name>
+                    <given>Unattended</given>
+                    <family>Access</family>
+                  </name>
+                </agentPerson>
+                <representedOrganization classCode="ORG" determinerCode="INSTANCE">
+                  <id extension="VNFKT" root="1.2.826.0.1285.0.1.10"/>
```

Comment:
The prescription was released to FER21, so should the author be at this organisation too?